cloud-tasks/app/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2019 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const createHttpTaskWithToken = require('./createTask');
+const express = require('express');
+
+const app = express();
+
+const QUEUE_NAME = process.env.QUEUE_NAME || 'my-queue';
+const QUEUE_LOCATION = process.env.QUEUE_LOCATION || 'us-central1';
+const FUNCTION_URL =
+  process.env.FUNCTION_URL ||
+  'https://<region>-<project_id>.cloudfunctions.net/sendPostcard';
```

Comment:
We should add TODOs if these are values that will not work unless the developer changes the source code.

Reply:
I'm going to remove these since I have the example values in other places.